google-chart.html

```diff
@@ -197,6 +199,25 @@
           this.options = {};
           this.rows = [];
           this.dataTable = null;
+
+          var head = document.querySelector('head');
+          var element = this;
+          var observer = new MutationObserver(function(mutations) {
+              mutations.forEach(function(mutation) {
+                for (var i = 0; i < mutation.addedNodes.length; i++) {
+                  var newNode = mutation.addedNodes[i];
+                  if (newNode.rel && newNode.rel == 'stylesheet') {
+                    //Add the stylesheet as an @import
+                    var cssImport = document.createElement('style');
+                    cssImport.innerHTML = "@import '" + newNode.href + "';";
+                    element.shadowRoot.appendChild(cssImport);
```

Comment:
> Hence my comment about better checking that the stylesheet that was added is actually from the origin you care about.

Ahh, yes, didn't see that comment...

If a known path that only applies to gviz stylesheets can work, I think the best approach would be to create a single style manager object that keeps track of what gviz styles exists in the DOM, and then the individual charts instances could query that info from the style manager on `attached` and subscribe to changes.